src/QueryBuilder/TableFilterQueryBuilder.php

```diff
@@ -0,0 +1,378 @@
+<?php
+
+namespace Keboola\StorageDriver\Teradata\QueryBuilder;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\DBAL\Query\QueryException;
+use Google\Protobuf\Internal\RepeatedField;
+use Keboola\Datatype\Definition\BaseType;
+use Keboola\Datatype\Definition\Teradata;
+use Keboola\StorageDriver\Command\Info\TableInfo;
+use Keboola\StorageDriver\Command\Table\ImportExportShared\DataType;
+use Keboola\StorageDriver\Command\Table\ImportExportShared\TableWhereFilter;
+use Keboola\StorageDriver\Command\Table\PreviewTableCommand;
+use Keboola\StorageDriver\Command\Table\PreviewTableCommand\PreviewTableOrderBy;
+use Keboola\StorageDriver\Shared\Utils\ProtobufHelper;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use LogicException;
+
+class TableFilterQueryBuilder
+{
+    public const OPERATOR_SINGLE_VALUE = [
+        TableWhereFilter\Operator::eq => '=',
```

Comment:
nemohl bys dat `use` na Operator at to tady neni vsude s tou relativni cestou?

Reply:
jo, pridano